google/cloud/aiplatform/models.py

```diff
@@ -3505,3 +3516,97 @@ def get_model_evaluation(
                 evaluation_name=evaluation_resource_name,
                 credentials=self.credentials,
             )
+
+    def evaluate(
+        self,
+        prediction_type: str,
+        target_column_name: str,
+        gcs_source_uris: List[str],
+        instances_format: str,
```

Comment:
I wonder if this could have been an instance of `Dataset`, because, this `model` has anyways been trained with one of those, such as

```
model = job.run(
   dataset=...  # instance of TabularDataset
   ...
)
```

so, we should know all the info so long as it has access to the `Dataset` this model was trained with.

Reply:
What if this `Model` *was* trained with a managed dataset? Do they know the underlying file blobs? Wouldn't evaluating w/ the underlining raw file blob, circumvent the whole point of `Dataset` (and lineage, etc.), regardless of how the `Model` was trained?

Speaking from the opposite side, I admit I haven't fully caught up to the latest story about `CustomTrainingJob`-trained `Model` and managed `Dataset` in Vertex AI ... may / can / should they be mixed? or never supposed to be mixed?